elements/pl-order-blocks/dag_checker.py

```diff
@@ -44,10 +68,10 @@ def check_grouping(submission, group_belonging):
         group_id = group_belonging.get(node)
         if group_id is not None and cur_group is None:
             cur_group = group_id
-            cur_group_size = 1
         elif group_id is None and cur_group is not None:
             return i
-        elif group_id is not None and cur_group is not None:
+
+        if group_id is not None and cur_group is not None:
```

Comment:
This was actually a hilarious bug--we were not grading correctly when there was a `pl-block-group` with only one answer in it. This is of course, a useless edge case, but I still find it funny that the bug was here for a year without anyone noticing.